src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicencePolicyIntegrationTest.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.integration
+
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType
+import java.nio.charset.StandardCharsets.UTF_8
+
+class LicencePolicyIntegrationTest : IntegrationTestBase() {
```

Comment:
I wrote this first and strictly asserted the json response, then moved over from the json to kotlin version so pretty confident this works as before.